app/src/main/AndroidManifest.xml

```diff
@@ -9,16 +9,20 @@
     android:label="@string/app_name"
     android:roundIcon="@mipmap/ic_launcher_round"
     android:supportsRtl="true"
+    android:name=".application.OppiaApplication"
     android:theme="@style/OppiaTheme">
-    <activity android:name="org.oppia.app.HomeActivity">
+    <activity android:name=".HomeActivity">
```

Comment:
Let's not make this change since HomeActivity was moved to a subpackage in a downstream PR.

Reply:
merged develop code n updated changes